lib/crypto.js

```diff
@@ -134,7 +133,18 @@ exports.createCredentials = function(options, context) {
 
   if (options.cert) c.context.setCert(options.cert);
 
-  if (options.ciphers) c.context.setCiphers(options.ciphers);
+  if (options.ciphers) {
+    c.context.setCiphers(options.ciphers);
+  } else if (!(process._usingV1040Ciphers() && options.ciphers === undefined)) {
```

Comment:
@jasnell Per https://github.com/joyent/node/pull/23947#issuecomment-101849036, we could replace this with:
```
} else if (!(process._usingV1040Ciphers()) {
```
since we said that `null` and `undefined`would mean "use the default ciphers list`. That would also mean that the empty string would make node use the empty string unless the v0.10.40 legacy cipher is used, but that also sounds reasonable.